fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -121,3 +136,56 @@ private fun PrivateBrowsingDescriptionPreview() {
         )
     }
 }
+
+/**
+ * Total Private Browsing Mode description.
+ *
+ * @param onLearnMoreClick Invoked when the user clicks on the who can see my activity link.
+ */
+@Composable
+fun FeltPrivacyModeDescription(
+    onLearnMoreClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .wrapContentHeight(),
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layerAccentNonOpaque,
+        elevation = 0.dp,
+    ) {
+        Column(
+            modifier = Modifier.padding(PaddingValues(16.dp)),
+        ) {
+            Text(
+                text = stringResource(id = R.string.felt_privacy_desc_card_title),
+                style = FirefoxTheme.typography.headline7,
+                color = FirefoxTheme.colors.textPrimary,
+                lineHeight = 20.sp,
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            ClickableSubstringLink(
+                text = stringResource(id = R.string.felt_privacy_desc_card_subtitle),
+                textColor = FirefoxTheme.colors.textPrimary,
+                textStyle = FirefoxTheme.typography.body2,
+                linkTextColor = FirefoxTheme.colors.textPrimary,
+                linkTextDecoration = TextDecoration.Underline,
+                clickableStartIndex = 94,
+                clickableEndIndex = 131,
```

Comment:
We don't want to hardcode the indices here because they won't be correct for translated text.

Reply:
Here's an example of how we need to modify the string https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt#518.

We will need to split up "Who might be able to see my activity?" into a separate string and have the original string accept a parameter.